roman_photoz/regtest/conftest.py

```diff
@@ -0,0 +1,126 @@
+"""Pytest configuration and fixtures for regression tests."""
```

Comment:
How close to identical are conftest.py and regtestdata.py to the equivalents in romancal?  Is it possible for us to make romancal a test dependency and import them from there, for example?  I don't know if circular dependencies are a problem but if they are we shouldn't do that.

The r00001 file was added to data/ but it's not clear to me that it's used in the tests?  I originally imagined that the new integration test was going to run on that file but that's not right.